docker/README.md

```diff
@@ -208,4 +211,57 @@ healthcheck:
 ## Using jemalloc
 
 Jemalloc is embedded in the image and will be used instead of the default allocator.
-You can read about jemalloc by reading the Synapse [README](../README.md)
\ No newline at end of file
+You can read about jemalloc by reading the Synapse [README](../README.md)
+
+### Running all worker processes in a single container for testing
```

Comment:
I feel like this should be a separate file. This file is copied to https://hub.docker.com/r/matrixdotorg/synapse, and it's not really relevant to the main `matrixdotorg/synapse` image. (indeed, it would be rather confusing for people reading it at https://hub.docker.com/r/matrixdotorg/synapse)

Reply:
Oh indeed, I completely forgot that it ends up on Docker Hub. Good call.

Reply:
Although I'm struggling to come up with a good place to put it... Perhaps README-testing.md? Or maybe it should go in `docs/dev/` or CONTRIBUTING instead?

Reply:
I'd go with a `README-testing.md` in the docker dir, since it's mostly describing how the docker image works. We can always add references to it from elsewhere (eg it would be good to stick a comment in the the dockerfile itself).

Reply:
I've now done this.